samples/sensor/ccs811/README.rst

```diff
@@ -0,0 +1,81 @@
+.. _ccs811:
+
+CCS811 Indoor Air Quality Sensor
+################################
+
+Overview
+********
+
+The CCS811 digital gas sensor detects volatile organic compounds for
+indoor air quality measurements.  It estimates carbon dioxide where the
```

Comment:
How about:
```
The CCS811 digital gas sensor detects volatile organic compounds (VOCs) for
indoor air quality measurements. VOCs are often categorized as pollutants
and/or sensory irritants and can come from a variety of sources such as
construction materials (paint and carpet), machines (copiers and processors),
and even people (breathing and smoking).
```

Reply:
Updated, with addition of "estimated CO2" (since it's not actually measuring CO2).